buildtest/cli/report.py

```diff
@@ -874,29 +886,38 @@ def print_report_summary_output(
         color (str): An instance of a string class that tells print_report_summary what color the output should be printed in.
         detailed (bool, optional): Print detailed output of the report summary if ``buildtest report summary --detailed`` is specified
     """
-
-    console.print("Report File: ", report.reportfile())
-    console.print("Total Tests:", len(report.get_testids()))
-    console.print("Total Tests by Names: ", len(report.get_names()))
-    console.print("Number of buildspecs in report: ", len(report.get_buildspecs()))
+    consoleColor = checkColor(color)
+
+    console.print("Report File: ", report.reportfile(), style=consoleColor)
+    console.print("Total Tests:", len(report.get_testids()), style=consoleColor)
+    console.print("Total Tests by Names: ", len(report.get_names()), style=consoleColor)
+    console.print(
+        "Number of buildspecs in report: ",
+        len(report.get_buildspecs()),
+        style=consoleColor,
+    )
```

Comment:
as discussed can we not change the color of all text to the consoleColor

Reply:
Are you talking about the output of the method or do you want me to edit this doc string to be more accurate.

Reply:
Ok so I removed the color from this section